tests/common/widgets/range-input/options.ts

```diff
@@ -0,0 +1,220 @@
+import {
+  createMultiSearchResponse,
+  createSearchClient,
+  createSingleSearchResponse,
+} from '@instantsearch/mocks';
+import {
+  normalizeSnapshot as commonNormalizeSnapshot,
+  wait,
+} from '@instantsearch/testutils';
+import userEvent from '@testing-library/user-event';
+
+import type { RangeInputWidgetSetup } from '.';
+import type { TestOptions } from '../../common';
+
+function normalizeSnapshot(html: string) {
+  // React InstantSearch adds a noRefinement modifier on the list.
+  // @MAJOR: Remove this once React InstantSearch aligns with spec.
```

Comment:
Would it make sense to add the modifier in JS and Vue instead?

Reply:
We could, but to make it complete we would also need to introduce the `noRefinementRoot` className. Shouldn't we do this separately during our effort to unify?

Reply:
Yep I'm fine with this, thanks for looking!